### PR TITLE
Adding io hypershift tests

### PR DESCRIPTION
**Pull request by JoaoFula**

### CODE REVIEW

**Comment by JoaoFula** [comment]
/pj-rehearse periodic-ci-openshift-insights-operator-release-4.15-insights-operator-test-hypershift-periodic

**Comment by JoaoFula** [comment]
/pj-rehearse periodic-ci-openshift-insights-operator-release-4.15-insights-operator-test-hypershift-periodic

**Comment by JoaoFula** [comment]
/pj-rehearse periodic-ci-openshift-insights-operator-release-4.15-insights-operator-test-hypershift-periodic

**Comment by JoaoFula** [comment]
/pj-rehearse periodic-ci-openshift-insights-operator-release-4.15-insights-operator-test-hypershift-periodic

**Comment by JoaoFula** [comment]
/pj-rehearse periodic-ci-openshift-insights-operator-release-4.15-insights-operator-test-hypershift-periodic

File: ci-operator/config/openshift/insights-operator/openshift-insights-operator-release-4.15.yaml [hunk 1]
```diff
@@ -268,6 +268,51 @@ tests:
...
     workflow: ipi-gcp
+- as: insights-operator-test-hypershift-periodic
+  cron: 0 0 * * *
+  steps:
+    cluster_profile: aws-2
+    env:
+      HYPERSHIFT_BASE_DOMAIN: hypershift.aws-2.ci.openshift.org
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.15.19-multi
+    pre:
+    - chain: ipi-gcp-pre
```
Comment (jmguzik) [inline ci-operator/config/openshift/insights-operator/openshift-insights-operator-release-4.15.yaml:279]: gcp chanin in aws testcase is smelly

Comment (JoaoFula) [reply ci-operator/config/openshift/insights-operator/openshift-insights-operator-release-4.15.yaml:279]: A very good point. Changing it.

**Comment by JoaoFula** [comment]
/pj-rehearse periodic-ci-openshift-insights-operator-release-4.15-insights-operator-test-hypershift-periodic

**Pull request closed without merging**